tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -197,4 +197,56 @@ For example, `13796619446086334065/0/tidb-audit-2022-04-21T18-16-29.529.log`. In
 
 ## Disable audit logging
 
-If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off** .
\ No newline at end of file
+If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off** .
+
+## Audit log fields
+
+For each database event record in audit logs, TiDB provides the following fields:
+
+> **Note:**
+>
+> In the following tables, empty max length of a field means that the data type of this field has a well defined constant length (for example, 4 bytes for INTEGER).
+
+| Col # | Field name | TiDB data type | Max length | Description |
+|---|---|---|---|---|
+| 1 | N/A | N/A | N/A | Reserved for internal use |
+| 2 | N/A | N/A | N/A | Reserved for internal use |
+| 3 | N/A | N/A | N/A | Reserved for internal use |
+| 4 | ID       | INTEGER |  | Unique event ID  |
+| 5 | TIMESTAMP | TIMESTAMP |  | Time of event   |
+| 6 | EVENT_CLASS | VARCHAR | 15 | Event type     |
+| 7 | EVENT_SUBCLASS     | VARCHAR | 15 | Event subtype |
+| 8 | STATUS_CODE | INTEGER |  | Response status of the statement   |
+| 9 | COST_TIME | INTEGER |  | Time consumed by the statement    |
+| 10 | HOST | VARCHAR | 16 | Server IP    |
+| 11 | CLIENT_IP         | VARCHAR | 16 | Client IP   |
+| 12 | USER | VARCHAR | 17 | Login username    |
+| 13 | DATABASE | VARCHAR | 64 | Event-related database      |
+| 14 | TABLES | VARCHAR | 64 | Event-related table name          |
+| 15 | SQL_TEXT | VARCHAR |  | Masked SQL statement   |
```

Comment:
Why no max length for this field? If we don't know, I suggest using the max length of a varchar (64k bytes).

Reply:
Do you mean SQL_TEXT?

Reply:
For the audit log file, there is actually no limit for the length of 'SQL_TEXT'. The whole SQL statement could be logged in the file as long as it is accepted by TiDB server, so you could treat it as the limit of ['max_allowed_packet'](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_allowed_packet), which is 64MB by default.

But since we have a table defined here, so the maximum length of varchar(64K) LGTM.

Reply:
OK, I will use 64 KB in the table for 'SQL_TEXT'